ui/src/logs/containers/LogsPage.tsx

```diff
@@ -808,8 +798,7 @@ class LogsPage extends Component<Props, State> {
   }
 
   private handleBarClick = (time: string): void => {
-    const formattedTime = moment(time).toISOString()
-
+    const formattedTime = new Date(Date.parse(time)).toISOString()
```

Comment:
I think this case just be `new Date(time).toISOString()`

Reply:
👍